product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Optimizing subscriber-only groups
+navTitle: Optimizing
+description: 'Optimizing subscriber-only groups in EDB Postgres Distributed'
+---
+
+With PGD 5.6 and later, it's possible to optimize the topology of [subscriber-only groups](overview). 
+
+In this optimized topology, a small number of fully active nodes, the write leaders of the data groups, replicate changes to the group leaders of subscriber-only groups, these group leaders then replicate changes to the other members of its subscriber-only group.
+
+## Requirements for the optimized topology
+
+You can't enable this model if a cluster has any of the following:
+
+* Data-nodes which are directly members of the top-level group.
+* No data-node sub-groups.
+* No data-node sub-groups with proxy routing enabled.
+
+If this is the case, the nodes in a subscriber only groups automatically revert to the full mesh topology.
+
+To get the benefit of the new SO group and node replication, you must have your data-nodes in subgroups, with proxy routing enabled on the sub groups.
+
+## How the optimized topology works
+
+For clusters using groups for their data nodes, subscriber only groups can use a more efficient model which uses a subscriber-only group leader, similar to a write leader in PGD Proxies.
+
+Each subscriber-only group uses that group leader to replicate changes to other subscriber-only nodes in its group. The group leader acts as a replication proxy for incoming changes.
+
+The write leader nodes in other non-subscriber-only groups replicate changes to the group leader of the subscriber-only group. Other nodes only replicate within their data groups, both within their own group and in other data groups.
+
+## Subscriber-only group leaders
+
+With PGD 5.6 and later, each subscriber-only group gets assigned a group leader of its own. 
+This is because subscriber-only groups don't have a group Raft consensus mechanism of their own. 
+Instead, the cluster’s top level group uses its Raft consensus mechanism to handle selection of each subscriber-only group’s group leader.
+This group leader selection is on by default on PGD 5.6, regardless of the topology optimization settings.
+
+Group leaders in subscriber-only groups are regularly tested for connectivity and if unavailable, the voting nodes of top-level group select a new subscriber-only node from the subscriber-only group to become group leader. 
+The new group leader is then selected. 
+
+With optimized technology turned off, this election has no effect on the replication topology. Without the optimized topology, all nodes replicates changes to all other nodes in the  cluster.
+
+## Group leaders in the optimized topology
+
+ With the optimized topology enabled, only the group leader receives changes from other nodes in the cluster and it takes on the responsibility of replicating those changes to the other nodes in the subscriber-only group. 
```

Comment:
It is not clear which group leader is referred here.

Reply:
```suggestion
 With the optimized topology enabled, only the subscriber-only group's group leader receives changes from other nodes in the cluster and it takes on the responsibility of replicating those changes to the other nodes in the subscriber-only group. 
```